documentation/modules/post/windows/manage/install_python.md

```diff
@@ -0,0 +1,80 @@
+This module downloads an embeddable Python3 interpreter onto the target file system, granting pentesters access to a lightweight Python interpreter. This module does not require administrative privileges or user interaction with installation prompts.
+### Creating A Testing Environment
```

Comment:
This isn't named correctly for the section contents. Should probably be named `Tested Versions` or something like that

Reply:
Fixed